libmscore/property.h

```diff
@@ -284,6 +284,7 @@ enum class Pid {
       FONT_FACE,
       FONT_SIZE,
       FONT_STYLE,
+      USER_LINE_SPACING,
```

Comment:
(Maybe just LINE_SPACING ?)

Reply:
Yes

Reply:
Or `TEXT_LINE_SPACING`

Reply:
I agree USER_ is ugly there. 
TEXT_LINE_SPACING sounds good. Just want something to stay away from plain LINE_SPACING since  "lineSpacing"  is used at some places to read spacing from fontmetrics.